### ISSUE TITLE
Move sediment parameters into mo_param_bgc

### ISSUE THREAD

**Issue by jmaerz** [NorESMhub/BLOM#296]
Hi @TomasTorsvik and @JorgSchwinger , I am now working on moving the sediment parameters also into `mo_param_bgc` - once this is ready, Jörg can pretty the code which can then be merged into the `beyond-CMIP6` branch.
